JesusFilm/core
# feat: textfield rtl

# Description

Confirm that the Right to Left (RTL) functionalities are applied to the TextField component and the components that are also using the TextField component. This can be done or checked by adding an RTL visual test for the TextField and affected components

[- Link to Basecamp Todo](https://3.basecamp.com/3105655/buckets/29688713/todos/5413251317)

# How should this PR be QA Tested?

Visit the storybook link: https://612c2a83fdc2b2003a5c2eb7-ltomrsmknn.chromatic.com/?path=/story/journeys-ui-textfield--rtl
Check the stories of TextField, TextResponse, and SignUp components

- [x] it displays labels right-aligned in RTL journeys
- [x] it displays text value right-aligned in RTL journeys
- [x] it displays the error right align in RTL journeys
- [x] it displays the hint right aligned in RTL journeys
- [x] it doesn't break the style of the textfield when changing between LTR (LeftToRight) to RTL stories
- [x] it doesn't have changes to any of the LTR story

# Checklist
